docs/concepts/pdp/configuration.mdx

```diff
@@ -264,6 +264,31 @@ Default: `<green>{time}</green> | {process} | <blue>{name: <40}</blue>|<level>{l
 
 Change this parameter to change the log format (e.g. remove colors).
 
+#### PDP_DATA_UPDATE_CALLBACKS
+
+Default: `[]`
+
+List of data update callbacks to be called when data is updated. For example:
+```json
+[
+  {
+    "url": "http://example.com/callback",
+    "key": "example",  // optional
+    "config": {  // optional
+      "headers": {
+        "Authorization":  "Bearer token"
+      },
+      "method": "GET",
+      "data": {
+        "key": "value"
+      }
+    }
+  }
+]
+```
+The default config is: `{"method": "POST"}`.
+
+For more information, see the [CallbackEntry](https://github.com/permitio/opal/blob/master/packages/opal-common/opal_common/schemas/data.py#L123) interface.
```

Comment:
missing danger that having slow API receiving the callbacks might affect the PDP synchronization time